js/modules/WebAPIProvider/VocabularyProvider.js

```diff
@@ -43,7 +43,17 @@ define(function (require, exports) {
 			});
 			domainPromise.resolve(domains);
 		});
-	})
+	});
+
+  function getConceptRecordCount(identifiers) {
```

Comment:
I believe, this file should have been moved to `services` folder. Could you confirm with @johnSamilin and move the file if it is true

Reply:
I believe you'll have to merge/rebase this PR onto latest master in order to get the correct refactored code .